src/main/java/com/mcmoddev/alt/util/ALTFileUtils.java

```diff
@@ -113,4 +120,77 @@ public static void safeDeleteDirectory(File file) {
 		}
 	}
 
+	public static void copyFromResourceIfNotPresent(ResourceLocation value) {
+		Path base = Paths.get(AdditionalLootTables.getLootFolder().toString(), value.getResourceDomain()).normalize();
+		createDirectoryIfNotPresent( base );
+		File container = Loader.instance().getIndexedModList().get(value.getResourceDomain()).getSource();

```

Comment:
Damn, right - while I could get and pass the mod id in from the value thats in the IMC, I'm relying on a ResourceLocation put together by someone else.